sarah-schroeder-sd.md

```diff
@@ -0,0 +1,13 @@
+# San Diego, CA
+
+### Restaurants
+- Tiger Tiger
+- Oscar's Fish Tacos
```

Comment:
@sarah-schroeder Where's Oscar's? I haven't heard of this one. 😄  Can you add a taco recommendation here?